.cirrus.yml

```diff
@@ -0,0 +1,55 @@
+BUILD_TEST_TASK_TEMPLATE: &BUILD_TEST_TASK_TEMPLATE
+  arch_check_script:
+    - uname -am
+  test_script:
+    - python --version
+    - python -m pip install --upgrade pip
+    - python -m pip install -r requirements_dev.txt
+    - python -m flake8
+    - python -m pydocstyle pact
+    - python -m tox -e test
+    # - make examples
+
+linux_arm64_task: 
+  env:
+    matrix:
+      # - IMAGE: python:3.6-slim  # This works locally, with cirrus run, but fails in CI
+      - IMAGE: python:3.7-slim
+      - IMAGE: python:3.8-slim
+      - IMAGE: python:3.9-slim
+      - IMAGE: python:3.10-slim
+  arm_container:
+    image: $IMAGE
+  install_script:
+    - apt update --yes && apt install --yes gcc make
+  << : *BUILD_TEST_TASK_TEMPLATE
+
+
+macosx_arm64_task:
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-base:latest
+  env:
+    PATH: ${HOME}/.pyenv/shims:${PATH}
+    matrix:
+      - PYTHON: 3.6
+      - PYTHON: 3.7
+      - PYTHON: 3.8
+      - PYTHON: 3.9
+      - PYTHON: 3.10
+  install_script:
+    # Per the pyenv homebrew recommendations.
+    # https://github.com/pyenv/pyenv/wiki#suggested-build-environment
+    # - xcode-select --install  # Unnecessary on Cirrus
+    - brew update    
+    # - brew install openssl readline sqlite3 xz zlib 
+    - brew install pyenv
+    - pyenv install ${PYTHON}
+    - pyenv global ${PYTHON}
+    - pyenv rehash
+  ## To install rosetta
+    # - softwareupdate --install-rosetta --agree-to-license
```

Comment:
rosetta isn't installed by default, so this is using arm64 binaries, and would fail if using x86_64 without roseta